core/src/main/java/hudson/model/AdministrativeMonitor.java

```diff
@@ -183,7 +183,7 @@ public void doDisable(StaplerRequest req, StaplerResponse rsp) throws IOExceptio
 
     /**
      * Required permission to view this admin monitor.
-     * By default {@link Jenkins#ADMINISTER}, but {@link Jenkins#SYSTEM_READ} is also supported.
+     * By default {@link Jenkins#ADMINISTER}, but {@link Jenkins#SYSTEM_READ} or {@link Jenkins#MANAGE} are also supported.
```

Comment:
But you cannot declare a monitor to appear for users with _either_ `SYSTEM_READ` _or_ `MANAGE` (suppressing any fix GUI in the former case). This seems like a significant limitation, though better than requiring `ADMINISTER`.

Reply:
Implemented, but requires 2 separate methods.